packages/hash/shared/src/types.ts

```diff
@@ -1,8 +1,46 @@
-// The types exported here are expected to be branded.
+import {
+  EntityId,
+  extractEntityUuidFromEntityId as extractEntityUuidFromEntityIdSubGraph,
```

Comment:
Can we not capitalise graph here, `subgraph` is a single word :)

Reply:
aef0ef5ffe42b83cd9364e155474280232ee0714